InternTrack.Core.Api/Services/Foundations/Interns/InternService.cs

```diff
@@ -36,6 +36,20 @@ public partial class InternService : IInternService
             return await this.storageBroker.InsertInternAsync(intern);
         });
 
+        public ValueTask<Intern> ModifyInternAsync(Intern intern) =>
+        TryCatch(async () =>
+        {
+            ValidateInternOnModify(intern);
+
+            Intern maybeIntern =
+            await this.storageBroker.SelectInternByIdAsync(intern.Id);
```

Comment:
Tab

Reply:
Should I put extra tab space before lines 42 to 50?